site/_scss/layouts/devsite-theme.scss

```diff
@@ -1,9 +1,877 @@
+
+$LABEL_FONT: 700 0.875rem/1.719rem #{$TEXT_PRIMARY_FONT_FAMILY};
+$LABEL_LETTER_SPACING: 0.1rem;
+
+$ROW_PADDING: 3.5rem 0;
+$ROW_PADDING_MOBILE: 0 1.875rem;
+
+[theme="web-theme"] {
+  --devsite-background-4: #fff;
+  --devsite-background-color: #{$BG_PRIMARY};
+  --devsite-button-background: #{$BUTTON_BG};
+  --devsite-button-background-hover: #{$BUTTON_BG};
+  --devsite-button-border: #{$BUTTON_BORDER};
+  --devsite-button-border-hover: #{$BUTTON_BORDER};
+  --devsite-button-border-radius: #{$BUTTON_BORDER_RADIUS};
+  --devsite-button-color: #{$TEXT_LINK};
+  --devsite-button-font: #{$BUTTON_FONT};
+  --devsite-button-height: auto;
+  --devsite-button-letter-spacing: #{$BUTTON_LETTER_SPACING};
+  --devsite-button-line-height: #{$BUTTON_LINE_HEIGHT};
+  --devsite-button-margin: #{$BUTTON_MARGIN};
+  --devsite-button-padding: #{$BUTTON_PADDING};
+  --devsite-button-primary-background: #{$CHROME_BLUE};
+  --devsite-button-text-transform: none;
+  --devsite-buttons-margin: #{$BUTTONS_MARGIN};
+  --devsite-card-background: #{$BG_PRIMARY};
+  --devsite-card-border: none;
+  --devsite-card-border-radius: #{$CARD_BORDER_RADIUS};
+  --devsite-description-content-font: #{$DESCRIPTION_FONT};
+  --devsite-heading-color: #{$TEXT_PRIMARY};
+  --devsite-h1-font: #{$H1_FONT};
+  --devsite-h1-letter-spacing: #{$TEXT_LETTER_SPACING};
+  --devsite-h2-font: #{$H2_FONT};
+  --devsite-h2-letter-spacing: #{$H2_LETTER_SPACING};
+  --devsite-h2-margin: #{$H2_MARGIN};
+  --devsite-h3-font: #{H3_FONT};
+  --devsite-header-icon-size: #{$HEADER_ICON_SIZE};
+  --devsite-heading-color: #{$TEXT_PRIMARY};
+  --devsite-landing-row-group-margin: 0;
+  --devsite-landing-row-header-text-margin: 0 auto;
+  --devsite-landing-row-header-text-max-width: 57.8125rem;
+  --devsite-landing-row-padding: #{$ROW_PADDING};
+  --devsite-item-body-display: block;
+  --devsite-item-description-margin: #{$DESCRIPTION_MARGIN};
+  --devsite-item-gap: #{$CARD_GAP};
+  --devsite-item-text-padding: 0;
+  --devsite-label-font: #{$LABEL_FONT};
+  --devsite-label-letter-spacing: #{$LABEL_LETTER_SPACING};
+  --devsite-primary-font-family: #{$TEXT_PRIMARY_FONT_FAMILY};
+  --devsite-primary-text-color: #{$TEXT_PRIMARY};
+  --devsite-text-color: #{$TEXT_PRIMARY};
+  --link-color: #{$TEXT_PRIMARY};
+  --link-rgb-background: #{$BUTTON_BG};
+  --link-visited-color: #{$TEXT_PRIMARY};
+
+
+  @media (max-width: 591px) {
+    --devsite-button-padding: #{$BUTTON_PADDING_MOBILE};
+    --devsite-buttons-margin: #{$BUTTON_MARGIN_MOBILE};
+    --devsite-landing-group-padding: 0;
+    --devsite-landing-row-padding: #{$ROW_PADDING_MOBILE};
+
+  }
+
+  .material-icons {
+    background-color: $CHROME_BLUE;
+    border-radius: 100%;
+    color: #fff;
+    display: inline;
+    padding: 0.46875rem;
+    position: relative;
+    top: 0.5rem;
+  }
+
+  .dcc-blog-card {
+    --devsite-buttons-margin: 0.5rem 0 0 -0.75rem;
+    --devsite-item-gap: 0;
+
+    padding: 5rem 2.125rem 3rem;
+
+    span {
+      position: absolute;
```

Comment:
I assume you target a card label here. Please use the specific selector, rather than every span on the card (yes, this is this place where devsite- selector makes more sense than "span")

Reply:
@mamieorine, please target `.devsite-landing-row-item-labels` (the parent element of the span you are currently targeting) instead to avoid accidentally affecting any spans that might appear in that element in the future.

Reply:
@matthiasrohmer  I think you might reviewed the old file, we updated this selector a few days back. Please see the latest file [here](https://github.com/GoogleChrome/developer.chrome.com/blob/4956d82e2973630b36fd268f5e720413ce6baa7c/site/_scss/layouts/devsite-theme.scss. ). Thank you 😊